package.json

```diff
@@ -3,10 +3,15 @@
   "version": "0.1.0",
   "description": "Endeavors is a values & activities tracking app that lets you:",
   "main": "index.js",
+  "engine": {
+    "node": "^6.11.0",
+    "npm": "^3.10.10"
+  },
```

Comment:
this `engine` property allows us to specify what version of node/npm we're using (i'm using a different version for a work project, so its helpful to track what i need to load up)

if you wanna manage your versions quick i recommend `nvm` or `n` packages